### PR TITLE
Feature/fhirclient

### PR DESCRIPTION
**Pull request by erikwiffin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: python
-python: "3.4"
+python: "3.5"
```
Comment (erikwiffin) [inline .travis.yml]: fhirclient has some circular imports that apparently break in 3.4 but don't in 3.5. I didn't look into why, I just upgraded my python.

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,15 @@
 ## The following requirements were added by pip freeze:
 astroid==1.4.7
 click==6.6
+coverage==4.1
 docopt==0.6.2
+git+git://github.com/smart-on-fhir/client-py.git#egg=fhirclient
```
Comment (erikwiffin) [inline requirements.txt]: This will track master until my commits are deployed.

File: researchapp/services/logging.py [hunk 3]
```diff
@@ -23,7 +23,7 @@ def log(response):
     requests.post(es_url, data=json.dumps(payload))
```
Comment (erikwiffin) [inline researchapp/services/logging.py]: I didn't use grequests here, so logging is synchronous and could break things.

File: .travis.yml (hunk 1 above)
Comment (jmandel) [inline .travis.yml]: That looks like a non production version today. That makes me a little uncomfortable.

File: README.md [hunk 4]
```diff
@@ -13,3 +13,18 @@ pip install -r requirements.txt
 ```
 ./manage.py devserver
 ```
+
+## Running the testso
```
Comment (jmandel) [inline README.md]: Typo

File: researchapp/fixtures.yml [hunk 5]
```diff
@@ -30,14 +30,14 @@ class: !!python/name:researchapp.models.providers.OAuthClient
 args:
   client_id: research-app
   client_secret: demo-secret-s4s
-  scope: launch/patient patient/*.read offline_access
+  scope: patient/*.read offline_access
```
Comment (jmandel) [inline researchapp/fixtures.yml]: http://syncfor.science/api-calls/ indicates our clients should all use "launch/patient"  -  why this deletion?

File: .travis.yml (hunk 1 above)
Comment (erikwiffin) [inline .travis.yml]: Are you sure? Python 3.5.2 is the latest according to python.org, and 3.5.2 is what our Dockerfile uses anyways, so it's probably better that travis matches that than 3.4, which is what I have installed (outside docker) on my test machine.

File: researchapp/fixtures.yml (hunk 5 above)
Comment (erikwiffin) [inline researchapp/fixtures.yml]: fhirclient adds that automatically. https://github.com/smart-on-fhir/client-py/blob/43e9f12b2507c11e2afbe55fa2f6c4bcac7c68cb/fhirclient/client.py#L91

File: .travis.yml (hunk 1 above)
Comment (jmandel) [inline .travis.yml]: Cool -- I was just misled by 

image

**Pull request merged**